word_scripts/word_file_merger.py

```diff
@@ -0,0 +1,21 @@
+import wordfind_site
+
+path_to_save = '../app/src/main/assets/words/'
+file_prefix = 'words-length-'
+response = wordfind_site.wordFindSite()
+
```

Comment:
This code should be a function too. And your other python file should call this file because this is the generic word file merger so many other scripts could be written that are custom to a specific website but this file merger will always be the same and should be called by the python scripts for each specific site.